### PR TITLE
lkl: Add netperf TCP_RR and TCP_STREAM to make test

### PR DESCRIPTION
**Pull request by liuyuan10**
Tested by:
cd tools/lkl
make test

Signed-off-by: Yuan Liu liuyuan@google.com

---

This change is [Reviewable](https://reviewable.io/reviews/lkl/linux/152)

### CODE REVIEW

File: tools/lkl/tests/run_netperf.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+then
+    test_name=$4
+fi
+if [ $# -ge 5 ]
+then
+    echo "BAD NUMBER of INPUTS."
+    exit 1
+fi
+
+if [ $use_taskset -eq 0 ]
```
Comment (pscollins) [inline tools/lkl/tests/run_netperf.sh]: Nit: you may want `if [ -z $use_taskset ]` instead

File: tools/lkl/tests/run_netperf.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    sudo ip link set dev $LKL_HIJACK_NET_IFPARAMS down &> /dev/null || true
+    sudo ip tuntap del dev $LKL_HIJACK_NET_IFPARAMS mode tap &> /dev/null || true
+    clean
+}
+
+trap clean EXIT
+
+# LKL_HIJACK_NET_IFTYPE is not set, which means it's not called from
+# hijack-test.sh. Needs to set up things first.
+if [ -z ${LKL_HIJACK_NET_IFTYPE+x} ]
```
Comment (pscollins) [inline tools/lkl/tests/run_netperf.sh]: Why `+x`? Typo?

File: tools/lkl/tests/run_netperf.sh (hunk 1 above)
Comment (liuyuan10) [inline tools/lkl/tests/run_netperf.sh]: is -z used to check the length of string is 0? I find it doesn't work here

Comment (liuyuan10) [inline tools/lkl/tests/run_netperf.sh]: I do find it's not correct. changed to string comparison.

File: tools/lkl/tests/run_netperf.sh (hunk 2 above)
Comment (liuyuan10) [inline tools/lkl/tests/run_netperf.sh]: $(var+x) returns NULL if unset and "x" if set. It just covers the case $var="" but doesn't really necessary here.

File: tools/lkl/tests/run_netperf.sh (hunk 1 above)
Comment (pscollins) [inline tools/lkl/tests/run_netperf.sh]: Hmm, I guess it needs to be `if [[ -z $use_taskset ]]` or `if [ -z "$use_taskset" ]` ([from here](http://stackoverflow.com/questions/3427872/whats-the-difference-between-and-in-bash)). I thought that `-z` was the preferred way to do this but it seems your solution here and below [is the better one, anyway](http://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash) so nevermind.

**Comment by pscollins** [comment]
LGTM

**Pull request merged**